packages/performance/CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
 # Unreleased
+- [changed] Enable new performance monitoring users to onboard in real time.
```

Comment:
This is a bold release update.  May be "Update the transport mechanism of Fireperf events".

Reply:
It makes sense, updated release note to match the SDK change.